docker/dockerfile.slim

```
@@ -30,9 +30,10 @@ ENV DUNE_CONTROL_PATH=${DUMUXINSTALLPATH}/dune-common/dune.module:${DUMUXINSTALL
 ENV PYTHONPATH ${DUMUXINSTALLPATH}/dumux/bin/testing:${PYTHONPATH}
 
 # Compile minmum version of preCICE
-ARG PRECICEVERSION=v3.0.0
-ARG PRECICEUBUNTU=jammy
-RUN git clone --depth 1 -b ${PRECICEVERSION} https://github.com/precice/precice.git precice-${PRECICEVERSION} && cd precice-${PRECICEVERSION} && mkdir build && cd build && cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DPRECICE_FEATURE_MPI_COMMUNICATION=OFF -DPRECICE_FEATURE_PETSC_MAPPING=OFF -DPRECICE_FEATURE_PYTHON_ACTIONS=OFF -DCMAKE_INSTALL_PREFIX=/usr && make -j 4 && make test && make install && cd ../../ && rm -rf precice-${PRECICEVERSION}
+ARG PRECICEVERSION=3.0.0
+ARG PRECICEBRANCH=${PRECICEVERSION}
+RUN bash -c 'if [[ "$PRECICEVERSION" =~ ^[0-9]\.[0-9]\.[0-9]$ ]]; then PRECICEBRANCH="v${PRECICEVERSION}"; fi;\
```

```suggestion
RUN bash -c 'if [[ "$PRECICEVERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then PRECICEBRANCH="v${PRECICEVERSION}"; fi;\
```
Just in case a (sub-)version hits 10. Also in the second file.